functions/APIs/deals.js

```diff
@@ -0,0 +1,115 @@
+const { db } = require('../util/admin');
```

Comment:
I recommend formatting this with prettier using two-space indents to match the front-end code.